theia-full-docker/Dockerfile

```diff
@@ -139,6 +139,17 @@ RUN apt-get -y install ruby ruby-dev zlib1g-dev && \
     gem install solargraph
 
 
+# Rust
+ENV RUSTUP_HOME=/usr/local/rustup \
+    CARGO_HOME=/usr/local/cargo \
+    PATH=/usr/local/cargo/bin:$PATH \
+    USER=$USER
```

Comment:
Another small thing I noticed: it did not seem to cause an issue, but in the container, environment variable `$USER` is set, but to nothing. I think it's because $USER is not set in Dockerfile. 

If I unset the variable, then `cargo new hello-rust` fails. So I guess it needs to be set, even if the value is empty. Was this intended on your part or did you want it to have a specific value?

Reply:
done, also updated #209, also final thought I think `go get also will not work in this image` it's the same case with permission, I'll drop pr on it also so np.